src/picker/assets/scss/style.scss

```diff
@@ -47,6 +47,21 @@
       cursor: default;
     }
   }
+  .#{$prefix}daychange{
+    position: absolute;
+    vertical-align: middle;
+    background: #417cf4;
+    height: 100%;
+  }
+
+  .#{$prefix}daychange.#{$prefix}nextday{
+    right: -12px;
+  }
+  .#{$prefix}daychange.#{$prefix}previousday{
+    left: -13px;
+  }
```

Comment:
Please fix typo errors in these lines

Reply:
Thank you @farshadfakhar
Can you please fix `nextday` and `previousday` too?

Reply:
@talkhabi I'm so sorry because of these. I have fixed the problems now

Reply:
It's  okay, Thanks.